packages/browser-repl/src/components/shell.tsx

```diff
@@ -1,4 +1,6 @@
-import React, { Component } from 'react';
+/* eslint-disable jsx-a11y/no-static-element-interactions */
+/* eslint-disable jsx-a11y/click-events-have-key-events */
```

Comment:
nit: can we turn these off for the specific lines involved? That way we can avoid adding more instances of this by accident.